### ISSUE TITLE
Combining with basicauth

### ISSUE THREAD

**Issue by martindale** [filebrowser/filebrowser#214]
### Instructions (remove before submitting):

1. Are you asking for help with using Caddy or File Manager? Please use our forum instead: https://forum.caddyserver.com.
2. If you are filing a bug report, please answer the following questions.
3. If your issue is not a bug report, you do not need to use this template.
4. If not using with Caddy, ignore questions 1 and 2.

### 1. Have you downloaded File Manager from caddyserver.com? If yes, when have you done that? If no, and you are running a custom build, which is the revision of File Manager's repository?

Yes, as of today.

### 2. What is your entire Caddyfile?
```text
files.somehost.com {
  basicauth / foo bar

  filemanager / /media/somehost.com {
    no_auth
  }
}
```

### 3. What are you trying to do?
Use a single HTTP-based login for all users.

### 4. What did you expect to see?
A working install.

### 5. What did you see instead (give full error messages and/or log)?
Continuous prompts for HTTP username and password, as well as redirecting to `/login`, despite `no_auth`.

### 6. How can someone who is starting from scratch reproduce this behaviour as minimally as possible?
Use the above config.

**Comment by hacdias** [comment]
Sorry, will investigate this tommorow.

**Comment by hacdias** [comment]
Related to #204

**Comment by hacdias** [comment]
Fixed with 1.2.6 😄 You can download the new version from caddyserver.com!
